### ISSUE TITLE
Overcoming Classifier Imbalance for Long-tail Object Detection with Balanced Group Softmax

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#14]
# TL;DR
In object detection, performance is poor in minor categories . The method for the classification task for such data sets is incompatible with object detection and does not improve the accuracy such much. Therefore, they proposed a method to create groups according to the number of the data and create an "other" class within each group. The proposed method improves the accuracy of minor category classification.
![image](https://user-images.githubusercontent.com/24524018/89533286-29d2d680-d82e-11ea-9d6e-690d6e31638f.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2006.10408

# Submission Dates(yyyy/mm/dd)
2020/06/18

# Authors and institutions
Yu Li, Tao Wang, Bingyi Kang, Sheng Tang, Chunfeng Wang, Jintao Li, Jiashi Feng
- Key Laboratory of Intelligent Information Processing, Institute of Computing Technology, Chinese Academy of Sciences, Beijing, China 
- University of Chinese Academy of Sciences, Beijing, China
- Department of Electrical and Computer Engineering, National University of Singapore, Singapore
- Institute of Data Science, National University of Singapore, Singapore 

# Methods

# Results
The overall score as well as the minor class score (AP_r) in particular has improved.
![image](https://user-images.githubusercontent.com/24524018/89533270-22abc880-d82e-11ea-9427-5cf7e17a85bf.png)

# Comments
